src/sources/kubernetes_logs/mod.rs

```diff
@@ -82,13 +82,13 @@ const SELF_NODE_NAME_ENV_KEY: &str = "VECTOR_SELF_NODE_NAME";
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields, default)]
 pub struct Config {
-    /// Specifies the label selector to filter `Pod`s with, to be used in addition to the built-in `vector.dev/exclude` filter.
+    /// Specifies the label selector to filter `Pod`s with, to be used in addition to the built-in `exclude` filter.
     extra_label_selector: String,
 
-    /// Specifies the label selector to filter `Namespace`s with, to be used in  addition to the built-in `vector.dev/exclude` filter.
+    /// Specifies the label selector to filter `Namespace`s with, to be used in addition to the built-in `exclude` filter.
     extra_namespace_label_selector: String,
```

Comment:
I don't know if we are, or need to document what the built-in filter is somewhere else - but it's something to be aware of if we're removing it here.